docker/serverless/env-start-esp.py

```diff
@@ -62,11 +62,14 @@ def main():
     PLATFORM = "Cloud Run"
 
     # The command being run must be the 0th arg.
-    ARGS = [CMD, "--enable_backend_routing", "--compute_platform_override='{}'".format(PLATFORM)]
+    ARGS = [CMD, "--compute_platform_override='{}'".format(PLATFORM)]
```

Comment:
Sorry, we are not OK with this change.

Reply:
Even with it being added back [here](https://github.com/cloudendpoints/esp/pull/717/files#diff-b5b4182ce03e9cd04c63eaf105e44d78R111)?

Reply:
Well, you should check ESP_ARGS.

Reply:
There's just a final check of the ARGS variable after ESP_ARGS has been processed: https://github.com/cloudendpoints/esp/pull/717/files#diff-b5b4182ce03e9cd04c63eaf105e44d78R110

If you'd like me to alter this in some way, please let me know!

Reply:
This  change is only needed because I can't pass in the `--enable_grpc_backend_ssl` if `--enable_backend_routing` is enabled. Maybe if the backends.address protocol is set to grpc+ssl this can all be avoided? Or if you remove the restriction that ``--enable_grpc_backend_ssl` can't be used with `--enable_backend_routing` then this may not be required (assuming `--enable_backend_routing` functions like the `--backend` flag)

Reply:
Sorry, I misread your changes about requiring ESP_ARGS.  This line is after processing of ESP_ARGS,  the change is fine now.

But I have a bigger concern about not setting --enable_backend_routing for serverless image.  Google used to have a plan to do a drop-in replacement for all serverless image when next version of ESP is ready.  This flag is to make sure nobody is using the features not supported by next version.  That plan is on and off.  I don't know the final decision.  
To be safe, I like to keep it on, to not shot myself in the foot.

Reply:
Well it's not turned off by default, it's enabled by default still, it's only omitted if the --backend flag is used. Again, if we want to keep the original flag, can we make it so the grpc:// protocol supports SSL backends (how http and https are allowed), maybe by adding a new protocol grpc+ssl://? Or allow the --enable_grpc_backend_ssl flag to be used with --enable_backend_routing?

Reply:
Sorry,  I prefer you build your own docker image then.

Reply:
Well that's discouraging... There are multiple examples of using the serverless image with OpenAPI but none with gRPC, and such small changes are required to get this working. The serverless image is quite a misleading name in this case.

I hope you reconsider this decision in the future so developers won't have to make our own docker image for such a trivial change.